doc/admin/deployment.md

```diff
@@ -985,10 +985,29 @@ TODO: add instructions
 
 ### Systemd Integration
 
-A preliminary systemd script to manage the DAOS server is available
-under utils/system. That being said, this startup method is not
-supported yet since the current DAOS version still relies on PMIx for
-DAOS server wireup.
+Systemd support for daos_server is still experimental as it will start the
+daos_server and daos_io_server components in PMIXless mode which is still in
+development.
+
+DAOS Server can be started as a systemd service. The DAOS Server
+unit file is installed in the correct location when installing from RPMs.
+If you wish to use systemd with a development build you must copy the service
+file from utils/systemd to /usr/lib/systemd/system. Once the file is copied
+modify the ExecStart line to point to your in tree daos_server binary.
+
+Once the service file is installed you can start daos_server
+with the following commands:
+
+```
+systemctl enable daos-server
+systemctl start daos-server
+
+To check the component status use:
```

Comment:
Nit - probably want to break these descriptive lines out of the code-formatted blocks